src/core/Table/Table.md

```diff
@@ -217,7 +217,9 @@ const data = [
 
 Use the `enableRowSelection` to allow row selection via Checkboxes on the left hand side. Use the `onSelectedRowsChange()` prop to detect selection changes. The function returns the `id`s of selected rows.
 
-Also provide a `rowSelectionCheckboxLabel` to each row object to give an accessible label to the selection checkbox.
+Alternatively, you can use the `enableSingleRowSelection` prop to allow single row selection via RadioButton.
+
+Also provide a `rowSelectionCheckboxLabel` to each row object to give an accessible label to the selection Checkbox/RadioButton.
 
 You can control the selected rows programmatically by using the `controlledSelectedRowIds` prop as shown in the second example below.
```

Comment:
Should probably be "as shown in the third example below" after the new example addition.